src/fparser/two/Fortran2003.py

```diff
@@ -4313,19 +4313,22 @@ class Scalar_Char_Initialization_Expr(Base):
 
 class Primary(Base):  # R701
     """
```

Comment:
Could you please replace """ with ''' as that is what is being used in updated code.

Reply:
+1 on consistency within the project.

Are you aware that other projects use exactly the opposite convention (e.g. [numpy](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)), and does that make a difference for the overall ruling (i.e. outside of this PR)?

FWIW they have also adopted a much more human [friendly docstring convention](https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring) than the [sphinx one](https://pythonhosted.org/an_example_pypi_project/sphinx.html#function-definitions) - the numpydoc and napolean tools make these quite compelling options. 

As I say, no matter what, I'll get these fixed up to the project's conventions, but happy to help if you are open to long-term improvements of these conventions.

Reply:
No, I didn't realise other projects take a different approach. That is what we do in PSyclone, but I can't remember why this was chosen originally. Having a quick look at PEP257 I see that the suggestion is " rather than ' so we would seem to be at odds with the suggested standard. Having said that, I don't think it matters that much, certainly compared with the challenges we have getting fparser to be more robust, to have increased functionality, improved usability, etc.!

So, yes please change for consistency in this PR and we'll review the convention outside of the PR.

Regarding docstring conventions, we're happy with sphinx but would obviously be open to suggestions for improvements. Perhaps you could explain the benefits, as you see them, at a future telco?